### DISCUSSION TITLE
Microwave option

### DISCUSSION THREAD

**Discussion by Eijpie** [thoukydides/homebridge-homeconnect#146] [Ideas]
Currently, I can only control the oven function with the Homebridge integration. Is it possible to get the microwave added to this as well?

I have a Bosch HMG8764/35

**Comment by thoukydides** [comment]
Sorry, it doesn't look like the Home Connect API supports any microwave functionality:
https://api-docs.home-connect.com/programs-and-options?#oven

You could try requesting that functionality here:
https://developer.home-connect.com/support/contact

**Comment by thoukydides** [comment]
The Home Connect API added Microwave appliance support on 5th August 2025. Older devices are identified as Oven appliances.
